hudi-common/src/main/java/org/apache/hudi/common/bootstrap/index/BootstrapIndex.java

```diff
@@ -73,12 +73,12 @@ public final boolean useIndex() {
   /**
    * Check if bootstrap Index is present and ensures readable.
    */
-  protected abstract boolean isPresent();
+  public abstract boolean isPresent();
```

Comment:
I reverted this method to be protected. This should not be used directly by  clients.